collection-scripts/common.sh

```diff
@@ -0,0 +1,87 @@
+#!/bin/bash
+# shellcheck disable=SC2034,SC2086,SC2001,SC2068
+# be careful of 2068, oc is very sensitive with "" on variables with error error: there is no need to specify a resource type as a separate argument when passing arguments in resource/name form (e.g. 'oc get resource/<resource_name>' instead of 'oc get resource resource/<resource_name>'
+
+export DST_DIR="must-gather"
```

Comment:
Can it be overwrittable? `export DST_DIR="${DST_DIR:-must-gather}"`

Reply:
we can pass in this but i think normally for the support engineers they will look into must-gather folder only. 
unless user need to tell them this folder is the one we used to collect all logs, this could only make it more complex.
i would leave it as is for now, if needed can make a follow up enhancement PR for the suggestion.

Reply:
follow up in https://issues.redhat.com/browse/RHOAIENG-8809 feel free to update ticket if anything else can be added